src/net.rs

```diff
@@ -0,0 +1,29 @@
+use errors::DenoResult;
+use hyper::rt::{Future, Stream};
+use hyper::{Client, Uri};
+use tokio::runtime::current_thread::Runtime;
+
+// The CodeFetch message is used to load HTTP javascript resources and expects a
+// synchronous response, this utility method supports that.
+pub fn fetch_sync_string(module_name: &str) -> DenoResult<String> {
+  let url = module_name.parse::<Uri>().unwrap();
+  let client = Client::new();
+
+  // TODO Use Deno's RT
+  let mut rt = Runtime::new().unwrap();
+
+  let body = rt.block_on(
+    client
+      .get(url)
+      .and_then(move |response| response.into_body().concat2()),
```

Comment:
Do we need the move here? We're not borrowing anything.

Reply:
Thanks - fixed.